src/Server/ExchangeServer.java

```diff
@@ -348,6 +348,22 @@ private String processCommand(String user, String line, int connectionId, PrintW
                 out = "ASK_OUT DONE";
             }
             break;
+            case "HELP": {
+                out = "MY_CASH - output your current cash. \nOutput format: MY_CASH_OUT <cash> \n\n"

```

Comment:
This has multiple newlines for a single request. It will break most non-telnet clients